src/lib/shady-render.ts

```diff
@@ -244,6 +244,9 @@ export const render =
     (result: TemplateResult,
      container: Element|DocumentFragment|ShadowRoot,
      options: ShadyRenderOptions) => {
+      if (!options || !options.scopeName) {
+        throw new Error('The `scopeName` option is required.');
```

Comment:
Can you add a test for this exception?

Reply:
Added; I also fixed this conditional which would still throw if `options` wasn't an object.